src/clusterfuzz/_internal/bot/tasks/utasks/minimize_task.py

```diff
@@ -600,10 +600,20 @@ def utask_main(uworker_input):
 
 def utask_postprocess(output):
   """Postprocess in a trusted bot."""
-  if output.error is not None:
+  if output.error:
     uworker_handle_errors.handle(output, HANDLED_ERRORS)
     return
 
+  # Since uworker_input is set automagically, this should never be true
+  if not output.uworker_input:
+    raise errors.BadStateError
```

Comment:
I probably wouldn't even have this check, but I guess it's fine.. Also please end comments with a period.

Reply:
nah i dont feel strongly about it. removing!